goal_src/jak1/pc/progress-pc.gc

```diff
@@ -1937,9 +1937,12 @@
               (sound-play "starts-options")
               (sound-volume-off)
               (set! (-> *game-info* mode) 'play)
-              (initialize! *game-info* 'game (the-as game-save #f) "intro-start")
-              (set-master-mode 'game)
-              )
+              (if (= (-> *pc-settings* speedrunner-mode?) #t)
+                  ;; Start a new game differently if speedrunning mode is active
+                  (speedrun-start-full-game-run)
+                  ;; start the game normally
+                  (initialize! *game-info* 'game (the-as game-save #f) "intro-start"))
+              (set-master-mode 'game))
```

Comment:
~~This works for me locally, but is the call to set-master-mode needed? 
I can see it is called on line 321 of game-info.gc inside of the initialize! game-info method.
since we are passing the `'game` symbol~~

Ive looked in places where `(initialize! *game-info* 'game ` is used and Gavin also sets the master mode, so good by me 👍

Reply:
ye the original code does that.